src/pipelines/epidemiology/id_authority.py

```diff
@@ -130,3 +179,32 @@ def parse(self, sources: Dict[str, str], aux: Dict[str, DataFrame], **parse_opts
         with open(sources[0]) as fd:
             records = json.load(fd)["update"]["harian"]
         return DataFrame.from_records(_parse_records("ID", records))
+
+
+# pylint: disable=missing-class-docstring,abstract-method
+class IndonesiaLevel2DataSource(DataSource):
```

Comment:
I would move this to a different file to follow our convention. If a file is named `xx_authority` then it should contain *only* data sources which are authoritative. Since this is a third party data source, I suggest creating a separate file for it (prefixed with `id_` as per the naming convention).

Reply:
moved to id_andrafarm.py